Source/WebCore/crypto/keys/CryptoKeyEC.h

```diff
@@ -33,14 +33,16 @@
 
 #if OS(DARWIN) && !PLATFORM(GTK)
 #include "CommonCryptoUtilities.h"
+#include <wtf/Vector.h>
+
+typedef std::variant<CCECCryptorRef, Vector<uint8_t>>PlatformECKey;
```

Comment:
missing space before `PlatformECKey`. Also, we prefer `using` the typedef nowadays:
`using PlatformECKey = std::variant<CCECCryptorRef, Vector<uint8_t>>;`

Reply:
done

Reply:
changed